docs/installation/automatic-data-balancing.rst

```diff
@@ -3,108 +3,36 @@
 Automatic Data Balancing on Docker
 ==================================
 
-This tutorial runs Confluent Auto Data Balancing (ADB) on Kafka, which allows you to shift data to create an even workload across your cluster.  By the end of this tutorial, you will have successfully run Confluent Auto Data Balancing CLI to rebalance data after adding and removing brokers.
+This tutorial runs :ref:`Confluent Auto Data Balancing (ADB) on Kafka <rebalancer>`, which allows you to shift data to create an even workload across your cluster.  By the end of this tutorial, you will have successfully run Confluent Auto Data Balancing CLI to rebalance data after adding and removing brokers.
 
-.. include:: includes/docker-tutorials.rst
-    :start-line: 2
+.. include:: includes/start-download.rst
 
-#. Start the services by using the example Docker Compose file.  Navigate to ``cp-docker-images/examples/enterprise-kafka``, where it is located:
+#.  Clone the Docker images repository, checkout the |release| branch, and navigate to the directory with the multi-datacenter example.
```

Comment:
`navigate to the directory with the multi-datacenter example.` -> I don't think that's what you intended here for ADB?

Reply:
copy/paste error